### ISSUE TITLE
yowsup unable to receive messages

### ISSUE THREAD

**Issue by harpreetrekhi** [jlguardi/yowsup#54]
Hi,
I am sending messages from my smartphone to yowsup, message is not getting delivered to yowsup and there is no read receipt from yowsup.

Earlier when I send the message to yowsup, it gets received and read receipt is acknowledged.

When I start yowsup demos in debug mode, i can see messages I have sent from smartphone but they are still not ack by yowsup and futher messages sent are also not received/ack by yowsup.

I am guessing it is due to ACK issue, can any help me with this?

Logs:

```
pi@jigsaw:~/yowsup $ yowsup-cli demos -d -y -c config
DEBUG:yowsup.stacks.yowstack:Initializing stack
DEBUG:yowsup.stacks.yowstack:Constructed Network Layer
DEBUG:yowsup.stacks.yowstack:Constructed Stanza Regulator Layer
DEBUG:yowsup.stacks.yowstack:Constructed Crypt Layer
DEBUG:yowsup.stacks.yowstack:Constructed Coder Layer
DEBUG:yowsup.stacks.yowstack:Constructed Logger Layer
DEBUG:yowsup.stacks.yowstack:Constructed Axolotl Layer
DEBUG:yowsup.stacks.yowstack:Constructed Authentication Layer - Messages Layer - Receipt Layer - Ack Layer - Presence Layer - Ib Layer - Iq Layer - notification Ib Layer - Iq Layer - Chatstate Layer - call Layer - Groups Iq Layer - Media Layer - Privacy Layer - Profiles Layer
DEBUG:yowsup.stacks.yowstack:Constructed CLI Interface Layer
Yowsup Cli client
==================
Type /help for available commands

[offline]:/L
DEBUG:yowsup.layers.network.layer:Connecting to e1.whatsapp.net:443
DEBUG:yowsup.layers.logger.layer:tx:
<stream:features>
</stream:features>

DEBUG:yowsup.layers.logger.layer:tx:
<auth passive="false" user="917053017198" mechanism="WAUTH-2">
▒6C▒▒▒/[▒▒iD▒6cq▒▒o▒K▒▒J▒-▒e▒▒,⅕▒ou▒X▒▒▒▒cf▒
W?b▒▒컓?­Q
HEX:f7364380d0eb2f5b01eaea836944e0366371968d6fd74bf7eb4af1982dfda865c5fd2ce28595d66f75ba589e9abfa76366dd0ab1e4e70838007ef0d380ed0d1a57123f62199cacecbb933fc2ad51
</auth>

DEBUG:yowsup.layers.logger.layer:rx:
<stream:features>
</stream:features>

DEBUG:yowsup.layers.logger.layer:rx:
<success status="active" kind="free" creation="1456506351" expiration="4444444444" props="5" t="1460113461">
▒_▒/)▒▒k
▒q▒H▒▒#▒J
HEX:f25fcf2f29ba816b0ceb9571a04895a01723a84a
</success>

DEBUG:yowsup.layers.protocol_iq.layer:starting ping thread.
Auth: Logged in!
[connected]:DEBUG:yowsup.layers.logger.layer:rx:
<ib from="s.whatsapp.net">
<dirty timestamp="1460113461" type="groups">
</dirty>
</ib>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1459958617A15" t="1459958618">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958617A15" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1459958622A18" t="1459958623">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958622A18" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="2" from="919818801098@s.whatsapp.net" id="1459958622A18" t="1459958623">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1459958622A18" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="2" from="919818801098@s.whatsapp.net" id="1459958622A18" t="1459958623">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958622A18" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="2" from="919818801098@s.whatsapp.net" id="1459958627A21" t="1459958628">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958627A21" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="1" from="919818801098@s.whatsapp.net" id="1459958627A21" t="1459958628">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1459958627A21" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="1" from="919818801098@s.whatsapp.net" id="1459958627A21" t="1459958628">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958627A21" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="1" from="919818801098@s.whatsapp.net" id="1459958631A24" t="1459958633">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1459958631A24" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="1" from="919818801098@s.whatsapp.net" id="1459958631A24" t="1459958633">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958631A24" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="63151377976B3D713F" t="1459958646">
<enc type="msg" v="2">
3
▒@▒Rjl▒W▒▒_@▒▒▒ꜼHub1▒pd"▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100018002210e4840dda4087526a6ca35785c35f40b7a794ea9cbc487562
</enc>
</message>

[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[63151377976B3D713F] Hi
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="63151377976B3D713F" t="1459958646">
</receipt>

Message 63151377976B3D713F: Sent delivered receipt and Read
[connected]:
PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="D12CCF32F9095445AC" t="1459958647">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd" ▒8:▒+▒▒▒▒▒0▒l▒▒▒▒
▒ڱ▒▒▒▒x▒▒HaE▒*▒-^
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100118002220fe383a8c2ba89eeee0e0308b116cffd01fc7fb0ca5dab1aab5b2a87805ffcd48614501922aa82d5e
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[D12CCF32F9095445AC] Hep
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="D12CCF32F9095445AC" t="1459958647">
</receipt>

Message D12CCF32F9095445AC: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="63151377976B3D713F">
</ack>

PuTTYPuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="8F0E321E401E0B1A47" t="1459958650">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd"l\8▒֢U▒f▒▒▒ڢ▒5▒d
                                       ▒▒m▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a1002180022106c5c38bfd6a25506db66fff4c9daa2cb35c7640bd6c56d88
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[8F0E321E401E0B1A47] Help
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="8F0E321E401E0B1A47" t="1459958650">
</receipt>

Message 8F0E321E401E0B1A47: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="D12CCF32F9095445AC">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="B36A52F82209D31623" t="1459958652">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd" Ev▒UuS▒uY▒▒^▒▒}*\▒q_
׋▒C▒M▒▒_▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100318002220144576f05575040053ac755987e8a75efa9f8c18df7d2a5c168f715f0ad78bc41643df4dac895f99
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[B36A52F82209D31623] Exercise
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="B36A52F82209D31623" t="1459958652">
</receipt>

Message B36A52F82209D31623: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="8F0E321E401E0B1A47">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="D211271A7B14591798" t="1459958654">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd" ؅▒:▒=▒*^5▒▒▒P5▒$▒▒;▒▒▒▒▒▒▒ι▒!▒▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100418002220d885da3afe3ded992a5e1f35ebebd100155035de2412b0b33bcff8f2c9c5e7a0c9ceb98c1621d3ee
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[D211271A7B14591798] Photo
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="D211271A7B14591798" t="1459958654">
</receipt>

PuTTYMessage D211271A7B14591798: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="B36A52F82209D31623">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="80F7BAE856381F1365" t="1459958666">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd"▒▒▒7▒▒▒▒▒Hh▒+▒O▒k▒▒F#▒*U
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100518002210f4f2d037aaaaa180824868992bc04ff56be3e64623b12a55
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[80F7BAE856381F1365] Hi
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="80F7BAE856381F1365" t="1459958666">
</receipt>

Message 80F7BAE856381F1365: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="D211271A7B14591798">
</ack>

PuTTYPuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="B1B34B50070F20FE9B" t="1459958676">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd" ▒      ▒1▒"M▒s>▒fL▒*▒▒▒I▒▒▒5▒▒^}▒P.▒fM
:
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100618002220d3099b319722004d96733ee366a37f4cf22aa78b8749c6f3a5d0359dc85e7d9e502ea566064d0c3a
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[B1B34B50070F20FE9B] Yes
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="B1B34B50070F20FE9B" t="1459958676">
</receipt>

Message B1B34B50070F20FE9B: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="80F7BAE856381F1365">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="909CE3359333620EA1" t="1459958687">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd"Y▒▒
                           ?▒#▒f▒<▒q▒{ԑb▒7M
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100718002210599a8e0b3fe68923b266803c10947117b37bd4916295374d
</enc>
</message>

PuTTY
[919818801098@s.whatsapp.net(06-04-2016 21:34)]:[909CE3359333620EA1] Fjd
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="909CE3359333620EA1" t="1459958687">
</receipt>

Message 909CE3359333620EA1: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="B1B34B50070F20FE9B">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="3735107EBEC30F402A" t="1459958748">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒p"L▒▒'?▒▒X}5x▒Uw▒▒▒▒▒6
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a1008180022104cb5e3273ffdd0587d3578fca555779615fcf1c7ce08d736
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:35)]:[3735107EBEC30F402A] Cc
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="3735107EBEC30F402A" t="1459958748">
</receipt>

Message 3735107EBEC30F402A: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="909CE3359333620EA1">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="0" from="919818801098@s.whatsapp.net" id="1459958758-1" t="1459958759">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1459958758-1" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="3735107EBEC30F402A">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="0" from="919818801098@s.whatsapp.net" id="1459958758-1" t="1459958759">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1459958758-1" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="D8BF6799BD8DD1E82B" t="1459958762">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd "k▒▒▒)▒▒#▒▒fz   ▒▒k:
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100918002210056b18c9c7fc29f78a95b6f823911cfd667a09bea96b3a13
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:36)]:[D8BF6799BD8DD1E82B] Yh
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="D8BF6799BD8DD1E82B" t="1459958762">
</receipt>

Message D8BF6799BD8DD1E82B: Sent delivered receipt and Read
[connected]:
PuTTYPuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="F286A3562DC25EB83C" t="1459958808">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd
" {▒M2▒▒"p䶚ҫ▒}▒x▒E-旺▒'m▒▒?]▒d▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100a180022207b14ee4d3212e2f3220570e4b69a00d2aba07dfc78e3452de697bade276df08199a0a13f5dbe64ff
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:36)]:[F286A3562DC25EB83C] Hb
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="F286A3562DC25EB83C" t="1459958808">
</receipt>

Message F286A3562DC25EB83C: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="D8BF6799BD8DD1E82B">
</ack>

PuTTYPuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="083B28892154B08379" t="1459958831">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd
                       "▒&tE▒_r▒EH
▒▒̀݌                               ▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100b18002210f9267445ef5f72da4512f77f480bef0d11a69fcd80dd8c08
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:37)]:[083B28892154B08379] ??
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="083B28892154B08379" t="1459958831">
</receipt>

Message 083B28892154B08379: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="F286A3562DC25EB83C">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="645688243352FACD25" t="1459958922">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd
"▒f▒▒▒΁▒▒YU     Ya▒     ▒uJx$
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100c18002210fd66d1e617e5a7ce819bc95955095961b209f3754a0f7824
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:38)]:[645688243352FACD25] Hh
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="645688243352FACD25" t="1459958922">
</receipt>

Message 645688243352FACD25: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="083B28892154B08379">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="E850DFE9B108003641" t="1459958967">
<enc type="msg" v="2">
3
" ▒▒g▒(▒C▒x▒Z▒^wSW6▒▒6▒J▒)zT▒`▒▒▒m▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100d18002220d9dc679928e59543cd78d85a895e7753571836f1fa36e94aae297a548f6005e4becbf869086d8c1e
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:39)]:[E850DFE9B108003641] H
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="E850DFE9B108003641" t="1459958967">
</receipt>

Message E850DFE9B108003641: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="645688243352FACD25">
</ack>

PuTTYPuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="BB15A3EE9885AACE91" t="1459958973">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd"▒▒e&E▒@▒▒▒▒     ؒ▒QGو@:
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100e18002210e4d4652645d540e9e5019acb091ad892995147d988403a02
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:39)]:[BB15A3EE9885AACE91] Dd
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="BB15A3EE9885AACE91" t="1459958973">
</receipt>

Message BB15A3EE9885AACE91: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="E850DFE9B108003641">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="7A78A73ADD75898C12" t="1459958976">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd" %c▒▒▒JKy▒▒|?▒`r▒l▒▒▒▒o`g{▒Q▒▒]▒▒`:▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a100f18002220256398d0c24a4b79918c7c3f9c600172936cc4e7c11fa46f0e60677b8d5180f89d5d8acd601f3a9d
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:39)]:[7A78A73ADD75898C12] Sjsj
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="7A78A73ADD75898C12" t="1459958976">
</receipt>

Message 7A78A73ADD75898C12: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="BB15A3EE9885AACE91">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<message from="919818801098@s.whatsapp.net" notify="Harpreet" offline="0" type="text" id="CD58417F22814ADC5C" t="1459959029">
<enc type="msg" v="2">
3
!k-dZ▒▒▒k$▒}<▒▒▒}`▒1▒pd" ▒▒ȍ�▒{▒'kz▒x▒I2˔▒V▒▒▒aO▒▒▒▒▒A$▒1▒
HEX:330a21056b2d645acbe8104b08a86b24c67d3ca1cfe9937d60b516d008318b7064181b7a101018002220a39ac88d17c29ace7bf9276b7ae178bf493214cb949307569fa0f9614fbfa11e9abefe4124fb3199
</enc>
</message>


[919818801098@s.whatsapp.net(06-04-2016 21:40)]:[CD58417F22814ADC5C] C
DEBUG:yowsup.layers.logger.layer:tx:
<receipt to="919818801098@s.whatsapp.net" type="read" id="CD58417F22814ADC5C" t="1459959029">
</receipt>

Message CD58417F22814ADC5C: Sent delivered receipt and Read
[connected]:
DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="7A78A73ADD75898C12">
</ack>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="0" from="919818801098@s.whatsapp.net" id="1459960442A13" t="1459960444">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1459960442A13" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<ack type="read" from="919818801098@s.whatsapp.net" class="receipt" id="CD58417F22814ADC5C">
</ack>
```

**Comment by jlguardi** [comment]
You aren't using my last master due some Id's has format "1459960442A13" instead of "1459960442-13".

Update your code and reopen the issue if the problem persists.

**Comment by uraid** [comment]
If you still have a problem receiving messages, refer to this solved issue: #44

**Comment by harpreetrekhi** [comment]
Issue was resolved earlier when @jlguardi mentioned to use the updated master, now the same problem appears again... Even i downlaod the new master but still the problem is coming, I am sending message from smartphone to yowsup and it is not received by yowsup, also I am not blocked for 30mins, as I have waited more than 6-7 hrs

debug logs:

pi@jigsaw:~/yowsup $ yowsup-cli demos -y -d -c config
DEBUG:yowsup.stacks.yowstack:Initializing stack
DEBUG:yowsup.stacks.yowstack:Constructed Network Layer
DEBUG:yowsup.stacks.yowstack:Constructed Stanza Regulator Layer
DEBUG:yowsup.stacks.yowstack:Constructed Crypt Layer
DEBUG:yowsup.stacks.yowstack:Constructed Coder Layer
DEBUG:yowsup.stacks.yowstack:Constructed Logger Layer
DEBUG:yowsup.stacks.yowstack:Constructed Axolotl Layer
DEBUG:yowsup.stacks.yowstack:Constructed Authentication Layer - Messages Layer - Receipt Layer - Ack Layer - Presence Layer - Ib Layer - Iq Layer - notification Ib Layer - Iq Layer - Chatstate Layer - call Layer - Groups Iq Layer - Media Layer - Privacy Layer - Profiles Layer
DEBUG:yowsup.stacks.yowstack:Constructed CLI Interface Layer

# Yowsup Cli client

Type /help for available commands

DEBUG:yowsup.layers.network.layer:Connecting to e16.whatsapp.net:443
DEBUG:yowsup.layers.logger.layer:tx:
stream:features
/stream:features

DEBUG:yowsup.layers.logger.layer:tx:
<auth passive="false" user="917053017198" mechanism="WAUTH-2">

> ▒6T
> ▒▒EHG!▒t▒▒▒!▒▒ 2▒W]▒▒▒▒▒
> kb(▒▒]▒)(.WU▒▒M▒▒Y▒"▒▒▒Q▒a▒▒▒▒
> HEX:3ef1361d540ccedc1d45484721f7beb3741597bf902191ce18208c7f32fb575dccddc00303ace30c6b6228c196d55d9029282e1d06575519f1c64dcbdc591183221ecce7ca51c861c8db05d711fd
> </auth>

PuTTYDEBUG:yowsup.layers.logger.layer:rx:
stream:features
/stream:features

DEBUG:yowsup.layers.logger.layer:rx:
<success status="active" kind="free" creation="1456506351" expiration="4444444444" props="5" t="1460300536">
▒▒*▒(▒b)▒▒▒F▒▒^▒▒
HEX:a39c2a8028cf6229c0e5b815f18e46baaf5ef3f7
</success>

DEBUG:yowsup.layers.protocol_iq.layer:starting ping thread.
Auth: Logged in!

<ib from="s.whatsapp.net">
<dirty timestamp="1460300537" type="groups">
</dirty>
</ib>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277516A12" t="1460277517">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277516A12" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="3" from="919818801098@s.whatsapp.net" id="1460277516A12" t="1460277517">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1460277516A12" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277516A12" t="1460277517">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277516A12" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277522A15" t="1460277524">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277522A15" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="3" from="919818801098@s.whatsapp.net" id="1460277522A15" t="1460277524">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1460277522A15" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277522A15" t="1460277524">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277522A15" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277532A18" t="1460277533">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277532A18" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt offline="3" from="919818801098@s.whatsapp.net" id="1460277532A18" t="1460277533">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" id="1460277532A18" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277532A18" t="1460277533">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277532A18" class="receipt">
</ack>

DEBUG:yowsup.layers.logger.layer:rx:
<receipt type="read" offline="3" from="919818801098@s.whatsapp.net" id="1460277537A21" t="1460277538">
</receipt>

DEBUG:yowsup.layers.logger.layer:tx:
<ack to="919818801098@s.whatsapp.net" type="read" id="1460277537A21" class="receipt">
</ack>
